docs/installation-and-operation/running-the-metabase-jar-file.md

```diff
@@ -31,7 +31,11 @@ If you just want to try Metabase out, play around with Metabase, or just use Met
 
 The below instructions are the same as the quick start above, just with a little more context around each step.
 
-### 1. Install Java JRE
+### 1. Install pre-requisites
+
+Metabase needs ttf-dejavu, fontconfig and java-cacerts packages to run correctly. Please ensure those are installed before continuing.
```

Comment:
They should be mentioned by the generic names, rather than the distro-specific package names. For instance, `ttf-dejavu` is only for Debian/Ubuntu-based distros. On Fedora/CentOS/RedHat-family, DejaVu font is not under that name.

Reply:
I just tried to made it as generic as possible